java/query/src/test/java/sleeper/query/executor/QueryExecutorTest.java

```diff
@@ -484,7 +466,12 @@ public void shouldReturnCorrectDataWhenRecordsInMultipleFilesInMultiplePartition
         InstanceProperties instanceProperties = createInstanceProperties();
         TableProperties tableProperties = new TableProperties(instanceProperties);
         tableProperties.setSchema(schema);
-        StateStore stateStore = getStateStore(schema, Collections.singletonList(5L));
+        StateStore stateStore = inMemoryStateStoreWithPartitions(
+                new PartitionsBuilder(schema)
+                        .rootFirst("root")
+                        .splitToNewChildren("root", "left", "right", 5L)
+                        .buildList()
+        );
         Partition leftPartition = stateStore.getLeafPartitions().stream()
```

Comment:
The partitions can now be retrieved from a PartitionTree created by the builder, rather than querying the state store.